src/Microsoft.PowerShell.Commands.Utility/commands/utility/WriteProgressCmdlet.cs

```diff
@@ -46,66 +37,48 @@ public sealed class WriteProgressCommand : PSCmdlet
         public string Status { get; set; } = WriteProgressResourceStrings.Processing;
 
         /// <summary>
-        ///
         /// Uniquely identifies this activity for purposes of chaining subordinate activities.
-        ///
         /// </summary>
         /// <value></value>
```

Comment:
Suggest removing the empty <value></value> comments in this class.

Reply:
Fixed.